app/src/main/java/io/homeassistant/companion/android/settings/qs/views/ManageTilesView.kt

```diff
@@ -154,6 +155,17 @@ fun ManageTilesView(
                         }
                     }
                 }
+
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Text(
+                        text = stringResource(R.string.tile_vibrate),
+                        fontSize = 15.sp
+                    )
+                    Switch(
+                        checked = viewModel.selectedShouldVibrate,
+                        onCheckedChange = { viewModel.selectedShouldVibrate = it }
+                    )

```

Comment:
The switch's "thumb" lacks contrast when off and using dark mode:

![Setting 'Vibrate when clicked' with a switch, where the switch "thumb"/circle blends into the background](https://user-images.githubusercontent.com/8148535/205452779-2fa2a85b-407c-48fa-8200-df2a711338c7.png)

This is solved everywhere else by specifying a different color like this:
```kotlin
colors = SwitchDefaults.colors(uncheckedThumbColor = colorResource(commonR.color.colorSwitchUncheckedThumb))
```

(In light mode it is easier to see due to the shadow)

Reply:
Thank you that does look better!